packages/devtools_app/test/test_data/vm_service_object_tree.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
```

Comment:
put this file in a subfolder under test_data. Perhaps test_data/debugger/

Reply:
Done.